frontend/app/src/composables/blockchain/balances/index.ts

```diff
@@ -23,13 +23,14 @@ export const useBlockchainBalances = () => {
 
   const fetch = async (
     blockchain: Blockchain,
-    ignoreCache = false
+    ignoreCache = false,
+    force = false
   ): Promise<void> => {
     const { loading, setStatus, resetStatus, isFirstLoad } = useStatusUpdater(
       chainSection[blockchain]
     );
 
-    if (loading()) {
+    if (loading() && !force) {
```

Comment:
In which case would you need to force refresh balances when they are already refreshing?

Reply:
For token detection.
So it guaranteed that after token detection, the balance refreshing should always happens.

In the past there is a case that 
- balance refreshing is running, then token detection is called
- when token detection is done, it should called the `balance refreshing`, but canceled because the previous `balance refreshing` is still running.

Although in the UI I try to make sure that when you call balance refreshing, the token detection button is disabled, and vice versa, but I think this force mechanism is nice to have (?)

Reply:
@rotki/backend what happens if the frontend schedules a second balance refresh while one is already running?

@lukicenturi let's see if this is a problem or not for the backend, but from the frontend perspective, rather than forcing a second one while one is running I think either canceling the first one before starting the second or queueing the second one would be a better approach.